pkg/ddl/job_table.go

```diff
@@ -360,6 +301,8 @@ func (s *jobScheduler) startDispatch() error {
 	// TODO move waitSchemaSyncedController out of ddlCtx.
 	s.clearOnceMap()
 	s.mustReloadSchemas()
+
+	logRateLimiter := rate.NewLimiter(rate.Every(30*time.Second), 3)
```

Comment:
Do you think we should use zapcore.NewSamplerWithOptions like this?

https://github.com/pingcap/tidb/blob/be16d490485c58fa902daf7b947b53f4b1b0f26e/pkg/statistics/handle/logutil/logutil.go#L37-L56

I can add it in a separate PR or in this PR.